app/controllers/order_controller.rb

```diff
@@ -0,0 +1,31 @@
+class OrderController < ApplicationController
+
+  STANDARD = 2
+  PREMIUM = 5
+  VIP = 10
+
+  def total_price(order)
+    details = OrderDetail.where('order_id=' + order.id)
+    sum = 0
+    details.each { |detail|
+      book = Book.where('id=' + detail.book_id.to_s)
```

Comment:
Did you test this function? 

Where will return array[]
so here if you want to use book.price, you need to use `book.first`

Reply:
i'll check it again.